apps/modernization-ui/src/components/FormInputs/DatePickerInput.tsx

```diff
@@ -25,8 +25,14 @@ const inputFormat = /^[0-3]?[0-9]\/[0-3]?[0-9]\/[0-9]{4}$/;
 
 const matches = (value: string) => inputFormat.test(value);
 
+const yearLength = /^(19|20)\d{2}$/;
+
 const isValid = (value?: string) => !value || matches(value);
 
+const matchesYear = (value: string) => yearLength.test(value);
+
```

Comment:
Could be cleaner but not sure what makes an invalid date. This allows for 1890-2999 year value